privacyidea/static/components/dialogs/views/dialog.no.token.html

```diff
@@ -0,0 +1,47 @@
+<!-- Modal -->
+
+<!-- this dialog is displayed, if the logged in user is or role "user" and he has no tokens. -->
+
+<div class="modal fade lock-dialog" id="dialogNoToken" tabindex="-1"
+     role="dialog"
+     ng-show="dialogNoToken == true"
+     data-backdrop="static" data-keyboard="false"
+     aria-labelledby="myModalNoToken"
+     aria-hidden="true">
+    <div class="modal-dialog">
+
+        <div class="modal-content">
+            <div class="modal-header">
+                <h4 class="modal-title" id="myModalNoToken" translate
+                >Welcome</h4>
+            </div>
+
+            <div class="modal-body" ng-show="browserLanguage == 'en'">
+                privacyIDEA is a system to manage your tokens.
+                You currently have no token assigned.
+                You can manage your own tokens here - you can create new tokens, set the PIN of the tokens, disable tokens
```

Comment:
I guess, evaluating all the user permissions is a bit too much but we should at least mention, that the allowed actions could depend on the configuration.

Reply:
I assumed that, if this feature is used, with a 99.8% propability the admin would create his own template anyways. This is why I chose a very bad text! ;-)